keps/sig-api-machinery/3903-unknown-version-interoperability-proxy/README.md

```diff
@@ -254,14 +254,24 @@ the number if needed, but upgrades aren't that frequent and few resources are
 changed on releases, so these requests should not be common. We will count them
 with a metric.
 
+There could be a large volume of requests for a specific resource which might result in the identified apiserver to be unable to serve the proxied requests. Although this scenario should not occur too frequently, since resource types that have large request traffic should not undergo upgrades too often.
+
 TODO: security / cert stuff.
 
 ## Design Details
 
-TODO: specific API change (x2)
+Specific API change(s)
```

Comment:
Usually in sections like this in KEPs, people quote the api types (or part of them) and actually write in the new field more or less the way it should look.

Reply:
Does having a list of APIService make sense for the serviceableBy field? Or should it instead be a list of serverIds?

Reply:
Removed ServiceableBy[] clause, doesn't seem necessary to have, as discussed.